docs/connect/python.md

```
@@ -47,9 +47,25 @@ for record in result.all():
     print(record)
 ```
 
+[SQLAlchemy] is the Python SQL toolkit and Object Relational Mapper that
+gives application developers the full power and flexibility of SQL.
+
+Python-based {ref}`dataframe`
+and [ML](#machine-learning) frameworks, and a few [ETL](#etl)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)